examples/all-clusters-app/esp32/README.md

```diff
@@ -53,17 +53,30 @@ step. To install these components manually, follow these steps:
 Currently building in VSCode _and_ deploying from native is not supported, so
 make sure the IDF_PATH has been exported(See the manual setup steps above).
 
--   In the root of the example directory, sync the CHIP tree's submodules and
-    source `idf.sh`. Note: This does not have to be repeated for incremental
-    builds.
+-   Setting up the environment
 
-          $ source idf.sh
+To download and install packages run bootstrap.sh
+
+        $ cd ${HOME}/tools/esp-idf
+        $ . ./export.sh
```

Comment:
Do you also want to add `./install.sh` ?

Reply:
Please add this to other examples as well

Reply:
It is already mentioned while setting up `esp-idf`.

Reply:
Still it will be better if you add that. It is present in the current README.md

Reply:
Added.